### PR TITLE
Convert temperature-taking guide to a modal sheet.

### LINKED ISSUE #81: [BUG] Design critique of "How to take your temperature"
## Suggestions ##
- [ ] Does this need to be a modal?  With so much content, it seems like it should be its own screen.
- [ ] Top-align the numbers.  They are currently center-aligned.
- [ ] Remove the top padding from the scroll pane and add it to the bottom of the header.  The divider between the two is where the top of the scroll pane should be.
- [ ] Change "Click here for instructions" copy to "How to check your temperature"

## Screenshot ##
https://photos.google.com/share/AF1QipPNzbQ9YfdnsdI-nSG2WbrB9J5y35_mu5uOORQ4X2pNHiUvaFTdWdgB3X9Y6WxfBw/photo/AF1QipOoQx8dCWk57e26MOjg0LquyaFlkwjO6fmfqKz3?key=am52QmZoVTZqOWxxWURVVE9EdkJpYUM3LWxfdGlB

## Smartphone: ##
 - Device: Essential
 - OS: Android 10 QP1A.191005.014
 - App version: 6d80da8

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

Converted the temperature instructions into a modal bottom sheet.

I've top-aligned the numbers, adjusted some of the copy, made it have a "scroll more" indicator if the content doesn't fit, and made it into a modal bottom sheet.

Here's how it looks now:

image

## Related Issues

- Fixes https://github.com/coronavirus-diary/coronavirus-diary/issues/81

### CODE REVIEW

File: lib/src/ui/screens/checkup/steps/intro.dart [hunk 1]
```diff
@@ -63,7 +63,7 @@ class _IntroStepState extends State<IntroStep> {
                   padding: EdgeInsets.symmetric(horizontal: 40),
                   child: Text(
                     localizations.introStepTimeForYourCheckup,
-                    style: Theme.of(context).textTheme.title,
+                    style: Theme.of(context).textTheme.headline6,
```
Comment (HansMuller) [inline lib/src/ui/screens/checkup/steps/intro.dart]: Glad to see these changes. We've made it to 2018!

Comment (gspencergoog) [reply lib/src/ui/screens/checkup/steps/intro.dart]: Yeah, I made those hoping they'd stick, but I'm still not sure we're actually on the `beta` branch for reals.

Comment (gspencergoog) [reply lib/src/ui/screens/checkup/steps/intro.dart]: Nope, back to the dark ages.  We're shipping on `stable`.

**Review by dnfield — APPROVED** [review]
LGTM

**Pull request merged**